example/android/app/build.gradle

```diff
@@ -59,8 +60,7 @@ dependencies {
     testImplementation 'junit:junit:4.12'
     androidTestImplementation 'com.android.support.test:runner:1.0.2'
     androidTestImplementation 'com.android.support.test.espresso:espresso-core:3.0.2'
-    implementation 'ly.count.android:sdk:22.09.0'
-    implementation 'com.google.firebase:firebase-messaging:20.2.1'
+    implementation 'ly.count.android:sdk:22.02.1'
+    implementation 'com.google.firebase:firebase-messaging:20.0.0'
```

Comment:
what's the reason to switch to an older version of 'firebase-messaging'  i.e 20.0.0' instead of '20.2.1'